src/engine_registry.cpp

```diff
@@ -307,7 +307,7 @@ namespace wr
 		.m_compute_shader_handle = std::nullopt,
 		.m_root_signature_handle = root_signatures::basic,
 		.m_dsv_format = Format::D32_FLOAT,
-		.m_rtv_formats = { wr::Format::R16G16B16A16_FLOAT, wr::Format::R16G16B16A16_FLOAT, Format::R8G8B8A8_UNORM },
+		.m_rtv_formats = { Format::R16G16B16A16_FLOAT, Format::R16G16B16A16_FLOAT, Format::R8G8B8A8_UNORM},
```

Comment:
Merge with master first!, the formats have been changed on master.

Reply:
master is wrong should never have been merged, the entire class lives inside the wr namespace